packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -854,6 +855,10 @@ function completeWork(
           workInProgress.effectTag |= Update;
         }
       }
+      if (enableSuspenseCallback) {
+        // Always notify the callback
+        workInProgress.effectTag |= Update;
```

Comment:
Might be nice to only schedule this effect when a callback has actually been provided.